react/Modal.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { pick } from 'ramda'
+import classnames from 'classnames'
+import styles from './components/styles.css'
+import { IconClose } from 'vtex.store-icons'
+import BaseModal from './components/BaseModal'
+import { useCssHandles } from 'vtex.css-handles'
+import { BackdropMode } from './components/Backdrop'
+import ModalContent from './components/ModalContent'
+import ModalTitle, { TitleTag } from './components/ModalTitle'
+import { useModalState, useModalDispatch } from './components/ModalContext'
+import { useResponsiveValue, ResponsiveInput } from 'vtex.responsive-values'
+
+interface Props {
+  titleTag: TitleTag
+  blockClass?: string
+  title?: string | ResponsiveInput<string>
+  titlePadding?: number | ResponsiveInput<number>
+  contentPadding?: number | ResponsiveInput<number>
+  showCloseButton?: boolean | ResponsiveInput<boolean>
+  backdrop?: BackdropMode | ResponsiveInput<BackdropMode>
```

Comment:
⛏[Nitpick] You could use `MaybeResponsiveInput<T>` to type these attributes:
https://github.com/vtex-apps/responsive-values/blob/master/react/hooks/normalizeResponsiveInput.ts#L17

Reply:
Nice